test/.pytest.expect

```diff
@@ -0,0 +1,3 @@
+pytest-expect file v1
```

Comment:
This expect fail is here because of https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16543 which is not yet merged

Reply:
Is this PR functional to be merged with the current `sourcetype`? If so, can we add `test_k8s_objects_sourcetype` later instead?

Reply:
Currently sourcetypes render correctly except `watch` case. So the only test case which is failing now is the one checking 
```
    - name: events
      mode: watch
```
Yes, we can add it later. I deleted pytest expect and `("sourcetype", "kube:object:event", 1),` from `test_k8s_objects_sourcetype`